hl/tools/gif2h5/decompress.c

```diff
@@ -296,6 +296,10 @@ Decompress(GIFIMAGEDESC *GifImageDesc, GIFHEAD *GifHead)
              * Build the hash table on-the-fly. No table is stored in the
              * file.
              */
+            if (FreeCode >= 4096) {
```

Comment:
Prefix/Suffix buffer size is `4096 * sizeof(int)`, so this should probably be `4096 * sizeof(int)` too.

Reply:
In this case the check would be correct since `FreeCode` is an index value into the `int` arrays